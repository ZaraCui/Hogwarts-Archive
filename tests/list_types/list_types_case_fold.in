ADD COLLECTION tests/data/books_small.csv
ADD SPELLBOOK tests/data/books_partial.csv 2001
LIST TYPES
EXIT

