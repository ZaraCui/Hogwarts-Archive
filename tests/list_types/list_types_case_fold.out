user: 12 spellbooks successfully added.

user: Successfully added: Advanced Charms (Filius Flitwick).

user: Charm
Curse
Defense
Herbology
Potion
Transfiguration

user: Ending Archive process.
